libs/components/flyout/src/lib/modules/flyout/types/flyout-config.ts

```diff
@@ -11,6 +11,7 @@ export interface SkyFlyoutConfig {
    * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
    * The description typically includes text on the flyout but not on items that users
    * interact with, such as buttons and forms.
+   * For more information about the `aria-describedby` attribute, see the [WAI-ARIA Definitions of States and Properties](https://www.w3.org/TR/wai-aria/#aria-describedby)
```

Comment:
```suggestion
   * For more information about the `aria-describedby` attribute, see the [WAI-ARIA definition](https://www.w3.org/TR/wai-aria/#aria-describedby).
```
Just my 2 cents, but I'd tighten this up and drop the capitalization. (And FWIW, even if we keep the full text that you have in place now, I don't think we need to capitalize because these aren't proper nouns and we aren't linking to that title within the document but a child section deeper into the document.)